wallet/src/de/schildbach/wallet/ui/preference/PreferenceActivity.java

```diff
@@ -68,7 +68,7 @@ private void initToolbar()
     @Override
     public void onBuildHeaders(final List<Header> target) {
         loadHeadersFromResource(R.xml.preference_headers, target);
-        hidePersonalSettingsIfLocked(target);
+        //hidePersonalSettingsIfLocked(target);
```

Comment:
It should be commented?

Reply:
This should not be commented out in the PR.